content-test/selectors/selectors.test.js

```diff
@@ -0,0 +1,134 @@
+const {assert} = require("chai");
+const {connect} = require("react-redux");
+const React = require("react");
+const TestUtils = require("react-addons-test-utils");
+const dedupe = require("lib/dedupe");
+
+const {
+  justDispatch,
+  selectSpotlight,
+  dedupedSites,
+  SPOTLIGHT_LENGTH
+} = require("selectors/selectors");
+const {rawMockData, createMockProvider} = require("test/test-utils");
+
+const validSpotlightSite = {
+  "title": "man throws alligator in wendys wptv dnt cnn",
+  "url": "http://www.cnn.com/videos/tv/2016/02/09/man-throws-alligator-in-wendys-wptv-dnt.cnn",
+  "description": "A Florida man faces multiple charges for throwing an alligator through a Wendy's drive-thru window. CNN's affiliate WPTV reports.",
+  "lastVisitDate": 1456426160465,
+  "images": [{
+    "url": "http://i2.cdn.turner.com/cnnnext/dam/assets/160209053130-man-throws-alligator-in-wendys-wptv-dnt-00004611-large-169.jpg",
+    "height": 259,
+    "width": 460,
+    "entropy": 3.98714569089,
+    "size": 14757
+  }]
+};
+
+const fakeState = rawMockData;
+const EmptyComponent = React.createClass({render: () => (<div />)});
+
+function connectSelectorToComponent(selector, InnerComponent = EmptyComponent) {
+  const Provider = createMockProvider();
+  const Connected = connect(justDispatch)(InnerComponent);
+  const instance = TestUtils.renderIntoDocument(<Provider><Connected /></Provider>);
+  return TestUtils.findRenderedComponentWithType(instance, InnerComponent);
+}
+
+describe("selectors", () => {
+  describe("justDispatch", () => {
+    it("should return an empty state object", () => {
+      assert.deepEqual(justDispatch(fakeState), {});
+    });
+    it("should pass dispatch to inner element", () => {
+      const instance = connectSelectorToComponent(justDispatch);
+      assert.property(instance.props, "dispatch");
+      assert.isFunction(instance.props.dispatch);
+    });
+  });
+  describe("justDispatch", () => {
```

Comment:
this looks like a copy of the above test. copy paste fail or am I missing a difference?